.github/workflows/nightly.yml

```diff
@@ -437,6 +437,26 @@ jobs:
         uses: ./.github/actions/notify-slack
         with:
           token: ${{ secrets.ACTION_MONITORING_SLACK }}
+      - name: Test Drupal
+        if: always()
+        run: |
+          php -i|grep memory_limit
+          echo memory_limit=256M > /etc/php.d/drupal.ini
+          git clone https://git.drupalcode.org/project/drupal.git drupal --depth=1
+          cd drupal
+          git rev-parse HEAD
+          php /usr/bin/composer install --no-progress --ignore-platform-reqs
+          export ASAN_OPTIONS=exitcode=139
```

Comment:
Not your fault, but we could make `ASAN_OPTIONS` a global env var as it's always the same anyway.

Reply:
Gonna file separate PR for that